markdown/dev/reference/api/point/slope/en.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Point.slope()
+---
+
+The `Point.slope()` method returns slope of a line made by this Point and that Point.
+
+## Signature
+
+```js
+point.slope(a,b)
+```
+
+## Example
+
+<Example caption="An example of the Point.slope() method">
+```js
+({ Point, points, Path, paths, Snippet, snippets, part }) => {
```

Comment:
Can we make this show something? Like, draw a line between those point and put the slope as text on the line?